module/documents/actor/actor.mjs

```diff
@@ -2718,6 +2718,7 @@ export default class Actor5e extends SystemDocumentMixin(Actor) {
     result.rolls ??= [];
     for ( const item of this.items ) {
       if ( foundry.utils.getType(item.system.recoverUses) !== "function" ) continue;
+      rollData.item = {...item.system};
```

Comment:
You should at the very least use `item.getRollData()` rather than 'reconstruct' the method here.

Reply:
The reason I did it this way was to avoid having to call `Actor#getRollData()` for every single item - instead only inserting the item-specific data each time.

Reply:
I imagine it won't matter, that's how it was before. It's also a one-off method, not during, say, data preparation.

Reply:
Makes sense, and improves the readability. Done.